pkg/sink/kafka/admin.go

```diff
@@ -15,116 +15,26 @@ package kafka
 
 import (
 	"context"
-	"errors"
-	"io"
-	"net"
 	"strconv"
 	"strings"
-	"sync"
-	"syscall"
 
 	"github.com/Shopify/sarama"
+	"github.com/pingcap/errors"
 	"github.com/pingcap/log"
 	"github.com/pingcap/tiflow/cdc/model"
 	cerror "github.com/pingcap/tiflow/pkg/errors"
-	"github.com/pingcap/tiflow/pkg/retry"
 	"go.uber.org/zap"
 )
 
 type saramaAdminClient struct {
-	brokerEndpoints []string
-	config          *sarama.Config
-	changefeed      model.ChangeFeedID
+	changefeed model.ChangeFeedID
 
-	mu     sync.Mutex
 	client sarama.Client
 	admin  sarama.ClusterAdmin
 }
 
-const (
-	defaultRetryBackoff  = 20
-	defaultRetryMaxTries = 3
-)
-
-func newAdminClient(
-	brokerEndpoints []string,
-	config *sarama.Config,
-	changefeed model.ChangeFeedID,
-) (ClusterAdminClient, error) {
-	client, err := sarama.NewClient(brokerEndpoints, config)
-	if err != nil {
-		return nil, cerror.Trace(err)
-	}
-
-	admin, err := sarama.NewClusterAdminFromClient(client)
-	if err != nil {
-		return nil, err
-	}
-	return &saramaAdminClient{
-		client:          client,
-		admin:           admin,
-		brokerEndpoints: brokerEndpoints,
-		config:          config,
-		changefeed:      changefeed,
-	}, nil
-}
-
-func (a *saramaAdminClient) reset() error {
```

Comment:
Is the upper layer, which is sinkManager or ddlsink,  responsible for retry operations when an error occurs after the reset is removed?